src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/breadcrumbs.tsx

```diff
@@ -140,7 +140,7 @@ class BreadcrumbsContainer extends React.Component<Props, State> {
     );
 
     this.setState({
-      searchTerm,
+      searchTerm: searchTerm as string,
```

Comment:
Should you also cast the value to a string with `String()` or a template string. Right now you could be passing a number or bool into the state and hit a runtime error if integer methods are used on the value.

Reply:
Yeah I can cast it as well, I was trying to avoid making functional changes while making all these type changes. Doing `searchTerm: string | number | boolean` in the params then casting to string here helps it match the consumer signature, and making it equivalent to what it was before (if we already had a runtime error we would have run into it), but you're right, I'll switch it.